pkg/kubelet/kubelet.go

```diff
@@ -833,21 +833,23 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	klet.evictionManager = evictionManager
 	klet.admitHandlers.AddPodAdmitHandler(evictionAdmitHandler)
 
-	// add sysctl admission
-	runtimeSupport, err := sysctl.NewRuntimeAdmitHandler(klet.containerRuntime)
-	if err != nil {
-		return nil, err
-	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.Sysctls) {
```

Comment:
so in server side validation we reject every sysctl, in the kubelet we ignore them. Is this intentional?

Reply:
It's not intentional. I will fix that to reject on the kubelet side.

Reply:
Oh, this one is actually intentional.